ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/defendant/CCDDefendant.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.hmcts.cmc.ccd.domain.defendant;
+
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.hmcts.cmc.ccd.domain.CCDAddress;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPartyType;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPaymentIntention;
+import uk.gov.hmcts.cmc.ccd.domain.CCDTimelineEvent;
+import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
+import uk.gov.hmcts.cmc.ccd.domain.CCJRequest;
+import uk.gov.hmcts.cmc.ccd.domain.defendant.statementofmeans.CCDStatementOfMeans;
+import uk.gov.hmcts.cmc.ccd.domain.evidence.CCDEvidenceRow;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Value
+@Builder
+public class CCDDefendant {
+    private String letterHolderId;
+    private String defendantId;
+    private LocalDate responseDeadline;
+
+    private CCDPartyType claimantProvidedType;
+    private String claimantProvidedEmail;
+    private CCDAddress claimantProvidedServiceAddress;
+    private String claimantProvidedName;
+    private CCDAddress claimantProvidedAddress;
+    private CCDAddress claimantProvidedCorrespondenceAddress;
+    private LocalDate claimantProvidedDateOfBirth;
+    private String claimantProvidedContactPerson;
+    private String claimantProvidedCompaniesHouseNumber;
+    private String claimantProvidedTitle;
+    private String claimantProvidedBusinessName;
+
+    private String representativeOrganisationName;
+    private CCDAddress representativeOrganisationAddress;
+    private String representativeOrganisationPhone;
+    private String representativeOrganisationEmail;
+    private String representativeOrganisationDxAddress;
+
+    private CCDPartyType partyType;
+    private String partyTitle;
+    private String partyName;
+    private LocalDate partyDateOfBirth;
+    private String partyPhone;
+    private String partyEmail;
+    private CCDAddress partyAddress;
+    private CCDAddress partyCorrespondenceAddress;
+    private String partyBusinessName;
+    private String partyContactPerson;
+    private String partyCompaniesHouseNumber;
+
```

Comment:
This is correct. This is single blank line to group the related fields after flattened objects in CCD.
FYI - If we have more than one consecutive extra lines, that is bad formatting.
Please correct otherwise.